docs/encyclopedia/workflows.mdx

```diff
@@ -260,6 +260,7 @@ This feature is useful for Workflow Definition logic that needs to be updated bu
 - [How to patch Workflow code in Go](/develop/go/versioning#patching)
 - [How to patch Workflow code in Java](/develop/java/versioning#patching)
 - [How to patch Workflow code in Python](/develop/python/versioning#python-sdk-patching-api)
+- [How to patch Workflow code in PHP](/develop/python/versioning#php-sdk-patching-api)
```

Comment:
Invalid link (typo). Need to replace `python` with `php`

Reply:
Thanks. Fixed.